Spellcheckmain-errordetection.py

```diff
@@ -0,0 +1,48 @@
+#SpellChecker
+
+#Function fetching comparable words from dictionary
+def readdictionaryfile(dictionaryfilename):
+    dictionarywords = []
+    inputfile = open(dictionaryfilename, "r")
+    for line in inputfile:
+        word = line.strip()
+        dictionarywords.append(word)
+    inputfile.close()
+    return dictionarywords
+
+#Function fetching words/text for evaluation
+def readtextfile(textfilename):
+    words = []
+    inputfile = open(textfilename, "r")
+    for line in inputfile:
+        wordsonline = line.strip().split()
+        for word in wordsonline:
+            words.append(word.strip(".,!\":;?").lower())
+    inputfile.close()
+    return words
+
+#Dictionary & Text comparison function
+def finderrors(dictionarywords, textwords):
+    incorrectwords = []
+    for word in textwords:
+        if word not in dictionarywords:
+            incorrectwords.append(word)
+    return incorrectwords
+
+#Printing errors - Temporary function
+def printerrors(errorlist):
+    print("The misspelled words are: ")
+    for word in errorlist:
+        print(word)
+
+

```

Comment:
Great! This appears to mean the main function initiating the process to take the words and return the error list of words.

Reply:
👍